vm/systemSmartContracts/auction.go

```diff
@@ -1627,6 +1629,41 @@ func (s *stakingAuctionSC) getTotalStaked(args *vmcommon.ContractCallInput) vmco
 	return vmcommon.Ok
 }
 
+func (s *stakingAuctionSC) getTopUp(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
```

Comment:
I think we need to return some more data. TotalStake, NumberOfActiveNodes. 
Why do we want to consume gas here ?

Reply:
Was implemented just like the getTotalStake. Will remove the gas consumption, is useless (in all view functions). Regarding the rest of the data: from the initial discussions I've understood that the only 2 fields will need to be used: numEligible and total top up. Will fetch the rest of the required values, if needed.

Reply:
it would be much simpler to fetch more data right now. plus to rename it to something more general. getTotalStakeData()

Reply:
I know, it will be made just like that (and will accommodate everything we need). I want to do it in another PR as this one got pretty large.